app/service.py

```diff
@@ -44,53 +90,50 @@ async def startup():
 async def shutdown():
     await db.disconnect()
 
-
-@app.post("/api/v1/providers", response_model=schemas.APIKey)
+@app.post("/api/v1/providers", response_model=schemas.APIKey, tags=["Requests with apiKey 🔑"])
 async def create_provider(
     provider: schemas.Provider, key: APIKey = Depends(validate_master_key)
 ):
     key = await models.Provider.create_new_key(db, provider.provider)
     return schemas.APIKey(key=key)
 
 
-@app.get("/api/v1/providers")
+@app.get("/api/v1/providers", tags=["Requests with apiKey 🔑"])
```

Comment:
This is a good opportunity to differentiate the regular apiKey from the MASTER apiKey in our docs.

Reply:
Corregido se separan los apiKey de los master.
Se ordena para que se visualice ultimo los Master apikey quedando así:

![image](https://github.com/tchx84/linka/assets/57275367/c0f140c8-5645-43da-aeae-776c96a1072c)